lib/modules.dart

```diff
@@ -260,12 +260,10 @@ class PurchasedItem {
         'orderId: $orderId, '
 
         /// android specific
-        'dataAndroid: $dataAndroid, '
         'signatureAndroid: $signatureAndroid, '
         'isAcknowledgedAndroid: $isAcknowledgedAndroid, '
         'autoRenewingAndroid: $autoRenewingAndroid, '
         'purchaseStateAndroid: $purchaseStateAndroid, '
-        'originalJsonAndroid: $originalJsonAndroid, '
 
```

Comment:
not needed fields removed as those are the copy of transactionReceipt

Reply:
Sure you can remove it!